woof/code/asteroids-code.html

```diff
@@ -0,0 +1,611 @@
+<head>
+    <script src="https://cdn.rawgit.com/stevekrouse/WoofJS/1e9b9a5e25ec87307efbe17924a09c4e0865fe6e/woof.js"></script>
+    <script>
+
+  // Sets backdrop to color
+  setBackdropColor('black')
+
+  // Creates ship sprite
+  var ship = new Text({
+    text: () => ">",
+    size: 50,
+    color: "white",
+    angle: 90
+  })
+
+  // Creates invisible sprite that will allow ship to move forward while rotating.
+  var invisShip = new Text({
+    text: () => "",
+    size: 50,
+    showing: false,
+  })
+
+  // Creates speed variable
+  var speed = 0
+  
+  
+  // Increases speed when up arrow pressed and decreases speed when up arrow not pressed. Moves invisible ship by speed.
+  forever(() => {
```

Comment:
This is a very clear and clever way to build momentum into a sprite in woof. Great work!